Jenkinsfile

```diff
@@ -1334,9 +1360,22 @@ pipeline {
                                            fi
                                            tnodes=$(echo $NODELIST | cut -d ',' -f 1-9)
                                            clush -B -S -o '-i ci_key' -l root -w ${tnodes} \
-                                             "if [ -e /sys/class/net/ib0 ]; then ifconfig ib0; ifup ib0; fi"
-                                           clush -B -S -o '-i ci_key' -l root -w ${tnodes} \
-                                             "if [ -e /sys/class/net/ib1 ]; then ifconfig ib1; ifup ib1; fi"
+                                             "set -x
+                                              for i in 0 1; do
+                                                if [ -e /sys/class/net/ib\\\$i ]; then
+                                                  if ! ifconfig ib\\\$i | grep "inet "; then
+                                                    {
+                                                      echo \"Found interface ib\\\$i down after reboot on \\\$HOSTNAME\"
+                                                      systemctl status
+                                                      systemctl --failed
+                                                      journalctl -n 500
+                                                      ifconfig ib\\\$i
+                                                      cat /sys/class/net/ib\\\$i/mode
+                                                      ifup ib\\\$i
+                                                    } | mail -s \"Interface found down after reboot\" $OPERATIONS_EMAIL
+                                                  fi
+                                                fi
+                                              done"
```

Comment:
I wouldn't want to to hold this PR from landing because of it, but this kind of repetitiveness could be very easily DRYed out by putting this scriptlet into a groovy variable.  This of course makes future maintenance (which is quite likely given what it's for) easier, but also keeps the size of our Jenkins file down and from hitting the size limit.

Reply:
Welcome to the Matrix...
Unfortunately only matrix steps can be run in parallel to each other.

Reply:
That's not what I mean.  I mean simply doing:

```
def ib_rigmarole = """set -x
                                              for i in 0 1; do
                                                if [ -e /sys/class/net/ib\\\$i ]; then
                                                  if ! ifconfig ib\\\$i | grep "inet "; then
                                                    {
                                                      echo \"Found interface ib\\\$i down after reboot on \\\$HOSTNAME\"
                                                      systemctl status
                                                      systemctl --failed
                                                      journalctl -n 500
                                                      ifconfig ib\\\$i
                                                      cat /sys/class/net/ib\\\$i/mode
                                                      ifup ib\\\$i
                                                    } | mail -s \"Interface found down after reboot\" $OPERATIONS_EMAIL
                                                  fi
                                                fi
                                              done"""
```

and then later on:

```
                                           clush -B -S -o '-i ci_key' -l root -w ${tnodes} ib_rigmarole
```